doc/UsersGuide/CustomizingTheWorkflow/InputOutputFiles.rst

```diff
@@ -353,6 +355,8 @@ AWS S3 Data Buckets:
 * GDAS: https://registry.opendata.aws/noaa-gfs-bdp-pds/
 * HRRR: https://registry.opendata.aws/noaa-hrrr-pds/ (necessary fields for initializing available for dates 2015 and newer)
 * A list of the NOAA Open Data Dissemination (NODD) datasets can be found here: https://www.noaa.gov/nodd/datasets
+* RRFS - experimental data is available starting 02/01/2024 for deteministic forecasts out to 60 hours at 00, 06, 12, and 18 UTC, and out to 18 hours for other cycles. Earlier dates, from 05/01/2023 to 01/31/2024, may contain only forecasts at 00, 06, 12, 18 UTC; user needs to verify that data exist for needed dates.
+  https://noaa-rrfs-pds.s3.amazonaws.com/index.html#rrfs_a/
```

Comment:
@natalie-perlin This section is supposed to include a list of data sources we _don't_ manage. Is this an EPIC-established bucket?

Reply:
This is not an EPIC-managed bucket. The folks running the RRFS parallels on WCOSS deliver data to this bucket in near real time.